app/views/layouts/application.html.erb

```diff
@@ -1,16 +1,18 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
+    <title><%= full_title( yield(:title)) %></title>
```

Comment:
Nice extraction. Normally we would create a presenter to compute all this with simple methods you can call which we expose on a page. Again probably not necessary here.